apps/learner/src/routes/(app)/+page.svelte

```diff
@@ -1,12 +1,51 @@
 <script>
+  import FloatingPlayer from '$lib/components/FloatingPlayer.svelte';
   import LearningUnit from '$lib/components/LearningUnit.svelte';
+
+  let isFloatingPodcastVisible = $state(false);
```

Comment:
Should we rename to `isFloatingPlayerVisible`?

Reply:
Yes, for consistency.